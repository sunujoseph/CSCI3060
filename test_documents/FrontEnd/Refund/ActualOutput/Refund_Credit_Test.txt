Enter Username: 
Enter Buyer Username: 
Enter Seller Username: 
Enter Minimum Bid: 
Error: Invalid Input
