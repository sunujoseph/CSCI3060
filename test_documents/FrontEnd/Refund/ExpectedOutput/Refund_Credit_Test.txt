Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter buyer username: 
Enter seller username: 
Enter amount of credit: 
Error: Minimum Refund Cannot Be Negative